config/autoload/kanbanize.acceptance.php

```diff
@@ -8,37 +8,38 @@
 		'service_manager' => array(
 				'factories' => array(
 						'Kanbanize\KanbanizeService' => function ($locator) {
-							$api = new KanbanizeAPIMock([
-								'114' => [
-									'taskid'		=> 114,
-									'columnname'	=> KanbanizeTask::COLUMN_ACCEPTED
-								],
-								'115' => [
-									'taskid'		=> 115,
-									'columnname'	=> KanbanizeTask::COLUMN_COMPLETED
-								],
-								'116' => [
-									'taskid'		=> 116,
-									'columnname'	=> KanbanizeTask::COLUMN_ONGOING
-								],
-								'117' => [
-									'taskid'		=> 117,
-									'columnname'	=> KanbanizeTask::COLUMN_COMPLETED
-								],
-								'118' => [
-									'taskid'		=> 118,
-									'columnname'	=> KanbanizeTask::COLUMN_ONGOING
-								],
-								'119' => [
-									'taskid'		=> 119,
-									'columnname'	=> KanbanizeTask::COLUMN_COMPLETED
-								],
-								'120' => [
-									'taskid'		=> 120,
-									'columnname'	=> KanbanizeTask::COLUMN_ACCEPTED
-								]
-							]);
-							return new KanbanizeServiceImpl($api);
+// 							$api = new KanbanizeAPIMock([
```

Comment:
In realtà non sapevo se era meglio toglierle o commentarle. Nella classe KanbanizeTask erano definite delle costanti per i nomi delle colonne che deve avere la board su Kanbanize. Questo non vale più dato che ogni organizzazione potrà impostare la propria configurazione di Kanbanize